lib/auth/auth_test.go

```diff
@@ -1193,6 +1194,45 @@ func TestGenerateHostCertWithLocks(t *testing.T) {
 	require.NoError(t, err)
 }
 
+func TestNewWebSession(t *testing.T) {
+	t.Parallel()
+	ctx := context.Background()
```

Comment:
nit: might as well just inline `context.Background()` here like you do on line 1207.

Reply:
revised